sycl/test-e2e/Graph/RecordReplay/ext_oneapi_enqueue_functions_submit_with_event.cpp

```diff
@@ -52,8 +52,9 @@ int main() {
 
   auto GraphExec = Graph.finalize();
 
-  Queue.submit([&](handler &CGH) { CGH.ext_oneapi_graph(GraphExec); });
-  Queue.wait_and_throw();
+  exp_ext::submit_with_event(Queue, [&](handler &CGH) {
+    execute_graph(CGH, GraphExec);
```

Comment:
```suggestion
    exp_ext::execute_graph(CGH, GraphExec);
```
Will change it for now.